devDocs/startupShutdown.md

```diff
@@ -43,41 +57,35 @@
+* Will force the main loop to exit and close most wx Windows.
 * We subsequently run `triggerNVDAExit` to ensure that clean up code isn't missed, and pump the queue to execute it.
 * Changing to using a custom message, which would allow a custom handling (eg just `triggerNVDAExit`), requires compatibility with messaging older NVDA versions that are only aware of `WM_QUIT`.
 
-### Out of process: `wx.EVT_END_SESSION`
-- This is a [wxCloseEvent](https://docs.wxwidgets.org/3.0/classwx_close_event.html) triggered by a Windows session ending.
-- On `wx.EVT_END_SESSION`, we save the config and play the exit sound.
-- Other actions are not performed as we have limited time to perform an action for this event. With NVDA being killed off very last, Windows may include NVDA in the Block shutdown dialog, but the user won't be able to read it if we are shutting down.
-
-## Technical startup and exit notes
+### When exiting from `wx.EVT_END_SESSION`
+* This is a [wxCloseEvent](https://docs.wxwidgets.org/3.0/classwx_close_event.html) triggered by a Windows session ending.
+* On `wx.EVT_END_SESSION`, we save the config and play the exit sound.
+* Other actions are not performed as we have limited time to perform an action for this event. With NVDA being killed off very last, Windows may include NVDA in the Block shutdown dialog, but the user won't be able to read it if we are shutting down.
 
-## The Message Window
+### Replacing an existing NVDA instance
 
-So that a new NVDA process can end a running NVDA process, NVDA accepts [WM_QUIT](#Out-of-process-WM_QUIT) messages from other processes and creates a window that can be discovered.
-The Windows API allows us to find a process by searching for a named window.
-NVDA creates a custom Window for accepting messages named "NVDA".
-
-Starting up is blocked by checking for an existing NVDA window and sending `WM_QUIT` to the process.
+To ensure only one NVDA instance is running, we need to be able to find and end a running process.
+So that a new NVDA process can find a running NVDA process, a named [message window](https://docs.microsoft.com/en-us/windows/win32/learnwin32/creating-a-window#creating-the-window) is created.
+A running NVDA process can be found by using the Windows API to search for the named window.
+A new NVDA process searches for an existing NVDA window, and if it is detected, sends `WM_QUIT`.
 The message window is created late during the start up, and destroyed early in exit and is not perfectly indicative of whether or not an NVDA process is running.
 As such, we have a [MutEx](#MutEx) that ensures a newly started process blocks until any previous NVDA has finished exiting.
 
-## MutEx
+### MutEx
 
-Preventing multiple copies. Note detects unusual shutdown of prior instance via "abandoned" mutex.
+To confirm that another NVDA process is not running,
+a [MutEx](https://docs.microsoft.com/en-us/windows/win32/sync/mutex-objects) is owned by the NVDA process.

```

Comment:
At the risk of spelling out the obvious, perhaps add to this something to say:
- NVDA will be blocked from starting until it can acquire the mutex.
- If it can not acquire the mutex within a timeout, startup is aborted.